pkgs/tools/networking/pykms/default.nix

```diff
@@ -29,6 +29,29 @@ let
     fi
   '');
 
+  loggingPatch = 
```

Comment:
We usually don't inline patches inside nix.

We prefer to fetch them from a PR to upstream, or if that's not feasible, ship them as a patchfile next to the `default.nix`.

Reply:
Fetching patches from a PR seems fragile. If I make a PR upstream and reference it in .nix, what happens if this PR is deleted for some reason?

I can extract the patch into a separate file, though.

Reply:
Usually, referring to the patch via something like https://github.com/$upstreamOwner/$upstreamRepo/pull/$pr/commits/$commitID.patch should be stable, at least as long as the fork still has this commit on some branch. But I agree, moving to a separate file might also be okay.

Reply:
You can reference the commit instead of the PR. The commit should be immutable even if you delete the PR.

Reply:
@Mic92 And if I delete the branch?

Reply:
> @Mic92 And if I delete the branch?

Also in that case. This commit is no longer accessible by any branch but the link still works: https://github.com/Mic92/dotfiles/commit/18869fa0637168610b64efbdb94da758afb1b84b

Reply:
@Mic92 Thanks.